docs/src/main/asciidoc/security-csrf-prevention.adoc

```diff
@@ -9,7 +9,12 @@ include::_attributes.adoc[]
 
 https://owasp.org/www-community/attacks/csrf[Cross-Site Request Forgery (CSRF)] is an attack that forces an end user to execute unwanted actions on a web application in which they are currently authenticated.
 
-Quarkus Security provides a CSRF prevention feature which consists of a xref:resteasy-reactive.adoc[RESTEasy Reactive] server filter which creates and verifies CSRF tokens in `application/x-www-form-urlencoded` and `multipart/form-data` forms and a Qute HTML form parameter provider which supports the xref:qute-reference.adoc#injecting-beans-directly-in-templates[injection of CSRF tokens in Qute templates].
+Quarkus Security provides the https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html#double-submit-cookie[Double Submit Cookie CSRF prevention]. This prevention requires that the CSRF token is never directly exposed to scripts executed on the client-side. In this extension, the CSRF token is:
```

Comment:
@turing85 I'm not sure this is how I'd restructure it. `Double Submit Cookie` is one of the techniques in the CSRF prevention, as opposed to a concrete well known feature of its own. 

I'd just add a clarification how the feature works, something like `CSRF prevention feature implements a Double Submit Cookie technique...`

Reply:
@sberyozkin This was what I wanted to convey in this paragraph. What do you think about the following wording?

> Quarkus Security provides CSRF protecthion through the Double Submit Cookie [mitigation|prevention|techique]. This [mitigation|prevention|techique] requires ....

Reply:
@turing85 I'd like to propose:
```
Quarkus Security provides a CSRF prevention feature which implements a [Double Submit Cookie technique](link). This technique requires  ( as per your text which nicely explains what it requires)
```

And then conclude as you did in the PR but without `Thus,`

Reply:
@sberyozkin Updated the PR accordingly :upside_down_face:

Reply:
Good stuff, thanks @turing85

Reply:
@sberyozkin you're welcome :tada: